src/template/Menu/Menu.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Icon, Menu } from 'antd'
+
+import logo from '../../assets/images/logo.png'
+import '../scss/Menu.scss'
+import { SITE_NAME } from '../../common/constants'
```

Comment:
use the absolute import please, **common/constants** instead of **../../../../../../../../../../../../common/constants**